packages/terra-compact-interactive-list/tests/jest/CompactInteractiveList.test.jsx

```diff
@@ -64,10 +98,50 @@ describe('Compact Interactive List', () => {
       expect(list.props().style.maxWidth).toBeFalsy();
       // the width should be equal to the sum of all columns width multiplied by numberOfColumns
       expect(list.props().style.width).toEqual('560px');
+
+      // cells should have width, but no maximumWidth or minimumWidth properties.
+      const cellElements = wrapper.find('.cell');
+      expect(cellElements.at(0).props().style.width).toEqual('40px');
+      expect(cellElements.at(0).props().style.minWidth).toBeFalsy();
+      expect(cellElements.at(0).props().style.maxWidth).toBeFalsy();
+
+      expect(cellElements.at(1).props().style.width).toEqual('200px');
+      expect(cellElements.at(1).props().style.minWidth).toBeFalsy();
```

Comment:
@kenk2 Good catch. I made some changes in https://github.com/cerner/terra-framework/pull/1916/commits/29905e326656dc0a2c3fa6ea37998c203c9024bb so that css props are added conditionally only if exist and check in tests for them toBeUndefined() instead of toBeFalsy().